src/components/themed/WiredProgressBar.tsx

```diff
@@ -1,67 +1,39 @@
-import React, { useEffect, useState } from 'react'
+import React from 'react'
 import { Animated, Easing, View } from 'react-native'
 
+import { DONE_THRESHOLD } from '../../constants/WalletAndCurrencyConstants'
+import { useWatch } from '../../hooks/useWatch'
 import { useSelector } from '../../types/reactRedux'
 import { isKeysOnlyPlugin } from '../../util/CurrencyInfoHelpers'
 import { cacheStyles, Theme, useTheme } from '../services/ThemeContext'
 
-const SHOW_UNSYNCED = false
-const SHOW_UNSYNCED_RATIO = 0.9
-
+/**
+ * Shows a bar indicating the the number of wallets that are fully synced across
+ * the entire account.
+ */
 export const WiredProgressBar = () => {
-  const progress = useSelector(state => {
-    const { userPausedWalletsSet } = state.ui.settings
-    const walletsForProgress = state.ui.wallets.walletLoadingProgress
-    const walletIds = Object.keys(walletsForProgress)
-    if (walletIds.length === 0) return 0
-
-    let sum = 0
-    let numPausedWallets = 0
-    const unsyncedWallets: Array<{
-      id: string
-      name: string
-      plugin: string
-    }> = []
-    for (const walletId of walletIds) {
-      const wallet = state.core.account.currencyWallets[walletId]
-      if (wallet == null) return 0
-
-      const paused = userPausedWalletsSet?.has(walletId)
-      const keysOnly = isKeysOnlyPlugin(wallet.currencyInfo.pluginId)
-
-      if (paused === true || keysOnly) {
-        numPausedWallets++
-        continue
-      }
-
-      sum += walletsForProgress[walletId]
-      if (walletsForProgress[walletId] !== 1) {
-        unsyncedWallets.push({
-          id: walletId,
-          plugin: wallet.currencyInfo.pluginId,
-          name: wallet.name ?? 'NO_NAME'
-        })
-      }
-    }
-    const numRunningWallets = walletIds.length - numPausedWallets
+  const theme = useTheme()
+  const style = getStyles(theme)
 
-    let ratio = sum / numRunningWallets
-    if (ratio > 0.99999) ratio = 1
+  const account = useSelector(state => state.core.account)
+  const { userPausedWalletsSet } = useSelector(state => state.ui.settings)
```

Comment:
`const userPausedWalletsSet = useSelector(state => state.ui.settings.userPausedWalletsSet)`, so we don't get superfluous updates.